insights/specs/insights_archive.py

```diff
@@ -187,6 +187,7 @@ class InsightsArchiveSpecs(Specs):
     pcp_metrics = simple_file("insights_commands/curl_-s_http_..127.0.0.1_44322.metrics_--connect-timeout_5")
     pcs_quorum_status = simple_file("insights_commands/pcs_quorum_status")
     pcs_status = simple_file("insights_commands/pcs_status")
+    pmrep_metrics = simple_file("insights_commands/pmrep_-t_1s_-T_1s_network.interface.out.packets_network.interface.collisions_swap.pagesout -o csv")
```

Comment:
it should be `insights_commands/pmrep_-t_1s_-T_1s_network.interface.out.packets_network.interface.collisions_swap.pagesout_-o_csv`

you can use mangle command to get the correct one:

```
# mangle "pmrep -t 1s -T 1s network.interface.out.packets network.interface.collisions swap.pagesout -o csv"
insights_commands/pmrep_-t_1s_-T_1s_network.interface.out.packets_network.interface.collisions_swap.pagesout_-o_csv
```

Reply:
Thanks for noticing, it was missed while updating it, updated it now.